metadata-transformer/src/main/java/no/unit/nva/metadata/NoLowercaseDCPrefixInMetadataFix.java

```diff
@@ -0,0 +1,25 @@
+package no.unit.nva.metadata;
```

Comment:
Honestly, this entire commit is no more than a sketch; like you said, more rewrites would be needed to apply this validation rule. I mostly wanted to know if this was what you wanted... Personally, I suspect it might be easier to apply this in MetadataConverter somehow without getting validator mixed into it. What do you think?

Reply:
I had hoped that we could find some solution that would allow us to do away with the lowercase/uppercase tests, at that point it makes sense to include in this PR.

Reply:
Difficult for me as I was looking at the PR title

Reply:
Ah, sorry about that.